crates/p2p-network/client-key

```diff
@@ -0,0 +1 @@
+~�Qe��0b���4Rc$�Y�|GH�X�)'Vt
```

Comment:
Should we move the key under examples? We are not using it in the production right?

Reply:
Moved. Not used in production.